dom/chrome-webidl/moz.build

```diff
@@ -74,8 +74,7 @@ WEBIDL_FILES = [
     "SessionStoreUtils.webidl",
     "StructuredCloneHolder.webidl",
     "TelemetryStopwatch.webidl",
-    "UniFFIPointer.webidl",
-    "UniFFIRustCallResult.webidl",
+    "UniFFI.webidl",
```

Comment:
Should we keep `UniFFIRustCallResult.webidl`? (or alternatively move its contents to `UniFFI.webidl`?

Reply:
I actually did that, but just forgot to delete the old `.webidl` files.  They weren't even being used since they weren't listed in `moz.build`